### DISCUSSION TITLE
Include JS and CSS files external to the main NextJS directory

### DISCUSSION THREAD

**Discussion by jordymeow** [vercel/next.js#14348] [Help] [answered]
Hi,

I have been looking for the solution for hours, but couldn't find any. I am not trying to include another module, but just files. They are JS (React/JSX) and CSS files.

I have a repository that looks like this:

- **/common** _(with files used by app, and nextjs)_
- **/app** _(a JS app)_
- **/nextjs** _(the nextJS project)_

If I try to use the files in /common from /nextjs, I get this kind of errors: _You may need an appropriate loader to handle this file type [...]_. But of course, that was expected. 

I would like to avoid creating a _npm module_ for _common_ (I would need to use link, and suddenly I know new issues would pop up). I really would like to keep things as simple as possible.

Thanks a lot for your ideas and help.

**Comment by jamesmosier** [comment, marked as answer]
Unfortunately not. All files you want transpiled/bundled must be included in your project folder. Otherwise as you mentioned you could go down the road of something like the [with-yarn-workspaces](https://github.com/vercel/next.js/tree/canary/examples/with-yarn-workspaces) example where you have multiple packages linked.

**Reply by jordymeow** [reply]
Thanks a lot for your reply @jamesmosier. It makes my life a bit easier to know what there is no solution for this. I will do this with yarn workspaces.
